Library/ExternalSubscription.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+
+    /// <summary>
+    /// Represents external_subscriptions for accounts
+    /// </summary>
+    public class ExternalSubscription : RecurlyEntity
+    {
+        private string _accountCode;
+        public string AccountCode => _accountCode;
+
+        private Account _account;
+        public Account Account
+        {
+            get { return _account ?? (_account = Accounts.Get(_accountCode)); }
+            internal set { _account = value; }
+        }
+
+        public string ExternalObjectReference { get; private set; }
```

Comment:
same question About this being nested under `external_resource`

Reply:
Are you asking for me to write a new class so that it's `externalSubscription.ExternalResource.ExternalObjectReference`?

Reply:
You're right. That's the proper way to do it. That's been updated